package.json

```diff
@@ -32,5 +32,8 @@
     "remark-rehype": "^8.0.0",
     "remark-slug": "^6.0.0",
     "unified": "^9.2.1"
+  },
+  "devDependencies": {
+    "sass": "^1.32.8"
```

Comment:
Sass should be a regular dependency, just like the unified/remark/rehype stuff is.

Reply:
fixed